### ISSUE TITLE
Does not run on 32 bit version of windows

### ISSUE THREAD

**Issue by Draqen4** [smowton/incredimail-converter-reynardware#20]
It says it is not a valid 32 bit application. Is it possible to make this able to run on 32 bit? Thanks

**Comment by smowton** [comment]
It seems that both the installer and the installed executable are already 32-bit in CS Release 2 (see https://github.com/smowton/incredimail-converter-reynardware/releases) Could you walk me through exactly what you did to get this error?

**Comment by Draqen4** [comment]
I downloaded the latest IncredimailConverter-CSr2-Setup.exe and tried to run it on a system that was running Windows XP Home SP3 32bit, when i was presented with the error.

**Comment by smowton** [comment]
Ahh, it's possible that the stone-age Windows XP is your problem, rather than the 32-bit flavour in particular. Do you have access to a more modern OS (Win7 or later I think) where you could copy the Incredimail data for conversion?

**Comment by Draqen4** [comment]
Ahh yeah I thought that that may have been the problem. The system was having weird issues with Incredimail and the c++ and .net runtime libraries giving weird errors too so it wouldn't surprise me.
Unfortunately I don't have the system with me anymore as it was for a client, I ended up moving them over to Thunderbird instead but if they have any further issues I'll have a go at converting the backup I made using my own system. I didn't think about merging the backup across to my own system xD.

Thank you for the help though. Keep up the good work :)
